test/types/tuple/split-init/multipleInit.good

```diff
@@ -0,0 +1 @@
+(1, "hi)
```

Comment:
One is probably not the right number of quotation marks.

Reply:
Probably not... Thanks for catching that!